### DISCUSSION TITLE
error in node_modules\dotenv\lib\main.js

### DISCUSSION THREAD

**Discussion by PavatharaniV** [goatandsheep/react-native-dotenv#478] [Q&A]
**Error:**
Android Bundling failed 3001ms
The package at "node_modules\dotenv\lib\main.js" attempted to import the Node standard library module "path".
It failed because the native React runtime does not include the Node standard library.
Learn more: https://docs.expo.dev/workflow/using-libraries/#using-third-party-libraries

**Comment by goatandsheep** [comment]
can you verify that you installed this library as a development dependency?
